initializer/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+import io.wax911.emoji.buildSrc.Libraries
+
+plugins {
+    id("io.wax911.emojify")
+}
+
+android {
+    namespace = "io.wax911.emojify.initializer"
+}
+
+dependencies {
+    implementation(project(Libraries.AniTrend.Emojify.contract))
+    implementation(project(Libraries.AniTrend.Emojify.emojify))
```

Comment:
Not so great to depend on `Libraries.AniTrend.Emojify.emojify`. If you want to remove this dependency one solution would be to make EmojiManager an interface in contract so that it would depend on the interface instead of the real class. Otherwise we could say we keep if like this but user would implement directly `com.github.anitrend:initializer` instead of `com.github.anitrend:android-emojify`

Reply:
True, the idea behind this was because an `initializer` needs to know what to invoke to produce a valid instance of an `EmojiManager` -> `EmojiManager.create(context, serializer)` in addition it would be almost impossible for a consumer to only implement `:initializer` without adding `:emojify`

Reply:
We could also delete this module and let the user implement it on their own. Other than that the PR looks good to me :)